src/modules/Providers/ProviderRow.tsx

```diff
@@ -67,8 +67,15 @@ const TextWithIcon = ({ value, Icon }: { value: string; Icon: JSXElementConstruc
   </>
 );
 
-const ProviderLink = ({ value, entity }: CellProps) => (
-  <ResourceLink kind={entity.kind} name={value} namespace={entity?.namespace} />
+const ProviderLink = ({ value, entity, t }: CellProps) => (
+  <>
+    <ResourceLink kind={entity.kind} name={value} namespace={entity?.namespace} />{' '}
+    {!entity.url && (
```

Comment:
Testing for url is incorrect, it's possible to create a provider 
with no url that is not the default provider, the default provider
must be of type kubvirt and be owned by the operator

Reply:
@rszwajko hi, talked with @ahadas about this and it's ok to use empty host to identify the default provider, 

resolving this issue as "ok to use empty url"